rocAL/rocAL/include/decoders/image/decoder.h

```diff
@@ -28,25 +28,22 @@ THE SOFTWARE.
 #include "parameter_factory.h"
 #include "parameter_random_crop_decoder.h"
 
-enum class DecoderType
-{
-    TURBO_JPEG = 0,//!< Can only decode
-    FUSED_TURBO_JPEG = 1, //!< FOR PARTIAL DECODING
-    OPENCV_DEC = 2, //!< for back_up decoding
-    HW_JPEG_DEC  = 3,
-    SKIP_DECODE  = 4, //!< For skipping decoding in case of uncompressed data from reader
-    OVX_FFMPEG = 5,//!< Uses FFMPEG to decode video streams, can decode up to 4 video streams simultaneously
+enum class DecoderType {
+    TURBO_JPEG = 0,        //!< Can only decode
+    FUSED_TURBO_JPEG = 1,  //!< FOR PARTIAL DECODING
+    OPENCV_DEC = 2,        //!< for back_up decoding
+    HW_JPEG_DEC = 3,
+    SKIP_DECODE = 4,  //!< For skipping decoding in case of uncompressed data from reader
```

Comment:
@fiona-gladwin can you add space to align with comments on above lines

Reply:
Done